command/server/config.go

```diff
@@ -482,6 +485,7 @@ func DefaultConfig() *Config {
 			PasswordFile:        "",
 			AllowInsecureUnlock: false,
 			UseLightweightKDF:   false,
+			DisableBorWallet:    false,
```

Comment:
Should this be `true`?

Reply:
I am not really sure of this. I think keeping `true` would be an explicit behaviour. We should instead keep default as usual behaviour. @ferranbt what do you think?